include/wavm/WAVMWasmModule.h

```diff
@@ -24,8 +24,8 @@ std::vector<uint8_t> wavmCodegen(std::vector<uint8_t>& wasmBytes,
 template<class T>
 T unalignedWavmRead(WAVM::Runtime::Memory* memory, WAVM::Uptr offset)
 {
-    const std::byte* bytes =
```

Comment:
The interface for these functions is all `uint8_t` to be consistent with the rest of the codebase, so here we don't have a choice. Because the two aren't compatible, we would need to do a wholesale switch otherwise we'd have casts scattered around the place. 

In general changes like this are probably best done wholesale to make the testing and reviewing process simpler. Issu on the topic: https://github.com/faasm/faabric/issues/199